src/base/bootstrap-overrides/_base.scss

```diff
@@ -234,6 +234,7 @@ abbr[title] {
 	abbr[title] {
 		border-bottom: 0;
 		text-decoration: underline dotted;
+		text-decoration-skip-ink: none; // sass-lint:disable-line no-misspelled-properties
```

Comment:
Add a mention that CSS is already defined in Bootstrap 4

Reply:
IMO that's kind of weird, but I'll update this PR to add a comment along those lines.